lib/dynamoid_advanced_where/nodes/exists_node.rb

```diff
@@ -13,7 +13,11 @@ def initialize(term_node: , **args)
 
 
       def to_condition_expression
-         "attribute_exists(##{expression_prefix})"
+         "NOT(attribute_not_exists(##{expression_prefix}) or ##{expression_prefix} = :#{expression_prefix}V2)"
```

Comment:
what happened that we had to not-not

Reply:
Otherwise we'd have base of `attribute_exists(##{expression_prefix}) or ##{expression_prefix} = [some value that's not null]`

And NOT that doesn't give what's desired.